ui/eureka/eureka_translations.ts

```diff
@@ -11,6 +11,18 @@ export const bunnyLabel: LocaleText = {
   ko: '토끼',
 };
 
+const sec2time = (timeInMs: number) => {
```

Comment:
is this `ms2time` and not `sec2time`? or, maybe I misunderstand what "sec" means here?